skynet/bandwidth.go

```diff
@@ -5,41 +5,59 @@ const (
 	KiB = 1024
 	// MiB megabyte
 	MiB = 1024 * KiB
+	// GiB gigabyte
+	GiB = 1024 * MiB
+	// TiB terabyte
+	TiB = 1024 * GiB
 
 	// SizeBaseSector is the size of a base sector.
 	SizeBaseSector = 4 * MiB
 	// SizeChunk is the size of a chunk.
 	SizeChunk = 40 * MiB
 
-	// PriceBandwidthRegistryWrite the bandwidth cost of a single registry write
-	PriceBandwidthRegistryWrite = 5 * MiB
-	// PriceBandwidthRegistryRead the bandwidth cost of a single registry read
-	PriceBandwidthRegistryRead = MiB
+	// RedundancyBaseSector describes the base sector redundancy the portal is
+	// using. This is not freely configurable because we need database
+	// consistency.
+	RedundancyBaseSector = 10
+	// RedundancyBaseSector describes the redundancy of regular chunks the
+	// portal is using. This is not freely configurable because we need database
+	// consistency.
+	RedundancyChunk = 3
+	// RedundancyAPIDivisor is the value by which we divide the raw used storage
```

Comment:
Does this mean we're not making a difference between base sector redundancy vs fanout redundancy when it comes to signalling the storage amounts to the user? Just asking out of curiosity.

Reply:
1. Yes, this is exactly what it meant. The  idea was to give the user a fizzy value that's in the close vicinity of the real one but without all the complications of reality.
2. This is now obsolete and unused, as it caused a lot of trouble. We switched to counting the actual on-disk size of the file towards the user's quota and reporting only that. I forgot to clean up this thing, which I will do now.